# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: browser/components/places/content/bookmarksHistoryTooltip.inc.xhtml
# Commit: 0a3da2090e48
# Full Hash: 0a3da2090e4859473f52b585bcb39baae2876b28
# Author: Jared Wein <jwein@mozilla.com>
# Date: 2021-02-26 03:55:47
# Regressor Bug: 1665390
# File Overlap Count: 1
# Description:
#   Bug 1665390 - Style bookmarks toolbar and tabs toolbar tooltips. r=NeilDeakin,ntim
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D104257
# ==============================================================================

diff -r 176b788753e8 -r 0a3da2090e48 browser/components/places/content/bookmarksHistoryTooltip.inc.xhtml
--- a/browser/components/places/content/bookmarksHistoryTooltip.inc.xhtml	Thu Feb 25 20:47:15 2021 +0000
+++ b/browser/components/places/content/bookmarksHistoryTooltip.inc.xhtml	Thu Feb 25 20:47:54 2021 +0000
@@ -4,9 +4,11 @@
 
 <!-- Bookmarks and history tooltip -->
 <tooltip id="bhTooltip" noautohide="true"
+         class="places-tooltip"
+         position="after_start"
          onpopupshowing="return window.top.BookmarksEventHandler.fillInBHTooltip(document, event)">
-  <vbox id="bhTooltipTextBox" flex="1">
-    <description id="bhtTitleText" class="tooltip-label" />
-    <description id="bhtUrlText" crop="center" class="tooltip-label uri-element" />
+  <vbox class="places-tooltip-box" flex="1">
+    <description crop="center" class="tooltip-label places-tooltip-title"/>
+    <description crop="center" class="tooltip-label places-tooltip-uri uri-element"/>
   </vbox>
 </tooltip>